src/webui/src/components/public-child/PaiTrialChild.tsx

```diff
@@ -36,7 +37,10 @@ class PaiTrialChild extends React.Component<PaiTrialChildProps, {}> {
                                         trial stdout
                                     </a>
                                     :
-                                    <span>trial stdout: {logString}</span>
+                                    <LogPathChild
+                                        eachLogpath={logString}
+                                        logName="Trial stdout:"
+                                    />
```

Comment:
why the log of pai mode use 2 different components? can they use the same `PaiTrialLog` component?

Reply:
`logPathChild ` just confirm that log string is hyperlink. But `log` nested components will refactor next release.